scripts/components/factsheets/top.component.js

```diff
@@ -0,0 +1,56 @@
+
+import _ from 'lodash';
+import formatNumber from 'utils/changeNumber';
+
+import TopTemplate from 'ejs!templates/factsheets/top.ejs';
+
+import 'styles/components/factsheets/top.scss';
+
+export default class {
+
+  constructor(settings) {
+    this.el = settings.el;
+    this.data = settings.data;
+    this.title = settings.title;
+    this.targetLink = settings.targetLink;
+    this.unit = settings.unit;
+
+    this._parseData();
+  }
+
+  _parseData() {
+    let baseUrlLink;
+
+    switch (this.targetLink) {
+    case 'place':
+      baseUrlLink = '/factsheet-place.html?nodeId=';
+      break;
+    case 'actor':
+      baseUrlLink = '/factsheet-actor.html?nodeId=';
```

Comment:
I think you can make this simpler by simply using a template string (ie '/factsheet-${this.targetLink}.html?nodeId=')